cmd/reach/app/main.go

```diff
@@ -301,6 +301,52 @@ func Main(ctx context.Context, args []string, stdin io.Reader, stdout, stderr io
 							},
 						},
 					},
+					{
+						Name: "unpack",
```

Comment:
If we wanted to add another subcommand for doing things by candidate or modulename, where would we put those flags?